StripeIdentity/StripeIdentity/Source/Elements/IndividualFormElement.swift

```diff
@@ -66,11 +67,17 @@ final class IndividualFormElement: ContainerElement {
             )
         }
 
+        if let phoneNumberSectionElement = phoneNumberSectionElement{
+            let phoneNumberElement = phoneNumberSectionElement.elements[0] as! PhoneNumberElement
```

Comment:
is the force casting (!) necessary here? why not something like:
`if let phoneNumberSectionElement = phoneNumberSectionElement, let phoneNumberElement = phoneNumberSectionElement.elements[0] as? PhoneNumberElement {`

Reply:
Updated, nice to avoid force casting